packages/http-client-java/emitter/src/emitter.ts

```diff
@@ -90,6 +92,8 @@ const EmitterOptionsSchema: JSONSchemaType<EmitterOptions> = {
 
     "group-etag-headers": { type: "boolean", nullable: true },
 
+    "enable-subclient": { type: "boolean", nullable: true, default: false },
```

Comment:
Do we want to set the option to true in our test to see the impact?

Reply:
Yes, we'd plan to set it to true for the subclient related tests (and for unbranded too, if we have thoses).

Currently my cadl-ranch PR is not merged, so we don't have them now. (test case, the basic one, is written before the microsoft package, I would need to migrate it first; and we will need more test on advanced cases)